docs/vehicle/state/data.md

```diff
@@ -6,7 +6,12 @@ A rollup of all the `data_request` endpoints plus vehicle configuration.
 
 _Note:_ all `*_range` values are in miles, irrespective of GUI configuration.
 
+<details>
```

Comment:
Please don't use details sections. This should be visible without clicking.

Reply:
The responses are both very large and make it rather unpleasant to read through this part of the documentation, especially on mobile. The response is not as crucial as the endpoint and the request type, so it makes sense for it to be hidden. 

I can revert the change, but it doesn't make much sense from a readability standpoint.

Kind regards.

Reply:
There are folks, myself included, that want to see the full response body so we know what keys to look for when parsing out the data. It shouldn't be a separate click away, even if it's long.